OAuth/ResourceOwner/Auth0ResourceOwner.php

```diff
@@ -32,34 +32,6 @@ class Auth0ResourceOwner extends GenericOAuth2ResourceOwner
         'profilepicture' => 'picture',
     ];
 
-    /**
-     * {@inheritdoc}
-     */
-    protected function doGetTokenRequest($url, array $parameters = [])
-    {
-        $parameters['client_id'] = $this->options['client_id'];
-        $parameters['client_secret'] = $this->options['client_secret'];
-
-        return $this->httpRequest(
-            $url,
-            http_build_query($parameters, '', '&'),
-            $this->getRequestHeaders(),
-            'POST'
```

Comment:
With the removal of `Auth0ResourceOwner::doGetTokenRequest`, are we sure the request method is still a POST request?

Reply:
Yes. This is handled inside `AbstractResourceOwner::httpRequest` when content / parameters are provided (see: https://github.com/hwi/HWIOAuthBundle/blob/409991fab7bd2da5896dff0239eaf192237e72cf/OAuth/ResourceOwner/AbstractResourceOwner.php#L248).

I've also specifically added this to the tests to avoid regression. See https://github.com/hwi/HWIOAuthBundle/pull/1604/files#diff-f4dfdc89fcff2dcabffddcf91416ebbaR62 and https://github.com/hwi/HWIOAuthBundle/pull/1604/files#diff-f4dfdc89fcff2dcabffddcf91416ebbaR155-R159.